helm/blueapi/values.yaml

```
@@ -124,5 +124,17 @@ worker:
       enabled: False
       host: "graylog-log-target.diamond.ac.uk"
       port: 12232
+  oidc:
+    well_known_url: https://authn.diamond.ac.uk/realms/master/.well-known/openid-configuration
+    client_id: blueapi
 initContainer:
   enabled: false
+
+oauth2-proxy:
+  config:
+    existingSecret: "oauth2-proxy-secret"
+  configFile: |
+    email_domains = [ "*" ]
+    upstreams = [ "{{ include "blueapi.fullname" . }}" ]
+    skip_auth_routes = ["GET=^/config/oidc"]
+    provider = keycloak-oidc
```

```suggestion
  config:
    existingSecret: "oauth2-proxy-secret"
  configFile: |
    email_domains = [ "*" ]
    upstreams = [ "{{ include "blueapi.fullname" . }}" ]
    skip_auth_routes = ["GET=^/config/oidc"]
    skip_provider_button = true
    oidc_issuer_url = https://authn.diamond.ac.uk/realms/master
  ingress:
    enabled: true
    hosts:
      - ckk43848.diamond.ac.uk
    path: /
    pathType: Prefix
```

Possibly? I don't know the exact mapping from the alphaConfig that I have used to the old config file, and mix-and-matching with the Secret?